src/makielayout/blocks.jl

```diff
@@ -288,6 +288,27 @@ function block_defaults(::Type{B}, attribute_kwargs::Dict, scene::Union{Nothing,
     return attributes
 end
 
+MakieCore.__obj_name(::Type{<:Block}) = "block"
```

Comment:
I quite dislike `_funcname` and I like `__func_name` even less :D  
It seems like all of the `__` functions could be easily replaced as well by just `x isa Plot` or `attribute_names`?

Reply:
I wasn't sure about `isa` since I don't know what else there is besides `x isa Plot` and `x isa Block`. I could do `x isa Plot` though.

The `__valid_attributes` is because I didn't know if it made sense to just define `attribute_names(::Type{<:Block})`. Does it? It could replace `__valid_atributes(::Type{<:Block})` below

Reply:
I don't think I knew about `plotsym`. I'll look into all of this soon, thanks.